mibidata/tests/test_tiff.py

```diff
@@ -105,6 +105,22 @@ def test_sims_only(self):
         self.assertIsNone(label)
         self.assertEqual(image.data.dtype, np.float32)
 
+    def test_sims_selected_channels(self):
+        tiff.write(self.filename, self.float_image)
+        image = tiff.read(self.filename, inc_channels=CHANNELS[1:3])
+        print(image.channels)
+        expected_image = tiff.read(self.filename)
+        expected_image = expected_image.slice_image(CHANNELS[1:3])
+        self.assertEqual(expected_image.channels, image.channels)
+        self.assertEqual(expected_image.metadata(), image.metadata())
+        self.assertEqual(expected_image, image)
```

Comment:
I think the asserts for `channels` and `metadata` are redudant because `MibiImage.__eq__` already includes both of those properties in it's equality check.